### DISCUSSION TITLE
Is there some way to operate opensnitch remotely or via terminal (ssh) without manually creating config files?

### DISCUSSION THREAD

**Discussion by mariomadproductions** [evilsocket/opensnitch#602] [Q&A] [answered]
Is there some way to operate opensnitch remotely or via terminal (ssh) without manually creating config files? Partially asking because I'm not exactly sure if manually creating config files is even sensible.

**Comment by gustavo-iniguez-goya** [comment, marked as answer]
Hi @mariomadproductions , 

No yet. There's a tool to visualize the connections (#208), that at some moment I'll have to update to allow configuring the daemon from the command line, but I can't provide an ETA.

**Reply by mariomadproductions** [reply]
Thanks for the info!
